### ISSUE TITLE
SectionEd: Message entry (Mock-up)

### ISSUE THREAD

**Issue by a16konka** [HGustavs/LenaSYS#4898]
Make up different mock-ups to make the message entry eye-catching, eg. color and icons. The importance of the message is determined by a type that changes the look of the entry in the entry list.

**Comment by b16linra** [comment]
As mentioned during the seminar, some sort of ranking system indicating the importance for different messeges could be a possible solution, but we are also free do something else if we think of something that could work.

**Comment by b16linra** [comment]
This seems to be related to issue #4848.

**Comment by a14isshi** [comment]
I think 4849 was made before Henrik came over for a meeting, this was done after as Henrik stated he wanted mock-ups for this. 4849 can probably be removed as they both refer to the same issue.

**Comment by a14isshi** [comment]
@HGustavs I created different mock-ups. I only tried to create a logo to imply that this is a message as if the design is to different it would make the over-all design look very unprofessional. What information I want back from you is:

1) If the overall designs is according to you a good solution or not?
2) If any icon of the 3 i made is good and if so which one you liked?
3) If any background of the 3 I presented is good and if so which one?

The background and icon is therefore two separated questions and not bound to each other.

Mock-up 1: A logo I created used with the standard background.
![message-icon 1](https://user-images.githubusercontent.com/37794167/39514946-e83abb28-4df8-11e8-874c-bd9557431083.png)

Mock-up 2: A logo I created used with a darker background from the standard.
![message-icon 3](https://user-images.githubusercontent.com/37794167/39515198-a89194fa-4df9-11e8-8f6d-7aa80a3eec9b.png)

Mock-up 3: Here is the idea that there will exist two kind of messages, a solution to how to declare the message type is not yet finished. The two message types is "regular" and "more important" messages. The first image is logo "regular" I created with a standard background. 
![message-icon 2](https://user-images.githubusercontent.com/37794167/39515581-d3ab7c5e-4dfa-11e8-963c-58a50d107331.png)

Second image is a "more important" logo with a lighter background (this one I find more fitting for messages as this separates them a little but still uses the same kind of color). As this message is more important the text also uses thicker text-type. 
![message-icon important](https://user-images.githubusercontent.com/37794167/39515665-1775ca34-4dfb-11e8-8ac7-46cd994e7151.png)

These were the once I could come up with, if you liked the idea of different messages types but didn't like the icons I used for it the triangle "!" can be used on the other two icons as well.
